Cargo.toml

```diff
@@ -52,7 +52,7 @@ path = "examples/deferred/main.rs"
 name = "flowmap"
 path = "examples/flowmap/main.rs"
 
-[[target.x86_64-pc-linux-gnu.example]]
+[[example]]
```

Comment:
we need to disable it on windows somehow (see AppVeyor build)... if not possible through `Cargo.toml`, maybe we can make the example code empty for some `#[cfg()]` condition?